src/lib/array.jl

```diff
@@ -41,6 +41,15 @@ end
   return (_project(x, dx), map(_->nothing, inds)...)
 end
 
+using CUDA
+import NNlib
```

Comment:
these imports shouldn't be here

Reply:
I think that's what @jgreener64 was referring to with "This isn't currently suitable for merging since it adds dependencies..."

Reply:
ah sorry, missed that